### PR TITLE
CORTX-32948: OVA Release Instructions

### PR DESCRIPTION
**Pull request by mukul-seagate11**
### Describe your changes in brief
OVA Release Instructions as per latest release

Tested OVA instructions on 940 build as per v0.12.0 version which works as expected as per output in CORTX-32919

Test-1 (Before reboot):
```
[root@cortx-ova ~]# kubectl get pod -o wide -n cortx
NAME                             READY   STATUS    RESTARTS        AGE     IP                NODE                    NOMINATED NODE   READINESS GATES
cortx-consul-client-2hds6        1/1     Running   1 (4h57m ago)   5h17m   192.168.220.157   cortx-ova.seagate.com   <none>           <none>
cortx-consul-server-0            1/1     Running   1 (4h57m ago)   5h17m   192.168.220.152   cortx-ova.seagate.com   <none>           <none>
cortx-control-6bb49c5b68-z88h8   1/1     Running   1 (4h57m ago)   5h17m   192.168.220.149   cortx-ova.seagate.com   <none>           <none>
cortx-data-g0-0                  3/3     Running   3 (4h57m ago)   5h17m   192.168.220.148   cortx-ova.seagate.com   <none>           <none>
cortx-ha-65494bd5c9-fpxgv        3/3     Running   3 (4h57m ago)   5h17m   192.168.220.151   cortx-ova.seagate.com   <none>           <none>
cortx-kafka-0                    1/1     Running   1 (4h57m ago)   5h17m   192.168.220.155   cortx-ova.seagate.com   <none>           <none>
cortx-server-0                   2/2     Running   3 (19m ago)     5h17m   192.168.220.158   cortx-ova.seagate.com   <none>           <none>
cortx-zookeeper-0                1/1     Running   1 (4h57m ago)   5h17m   192.168.220.147   cortx-ova.seagate.com   <none>           <none>

[root@cortx-ova ~]# kubectl get svc cortx-server-0 -n cortx |grep NodePort
cortx-server-0 NodePort 10.99.255.213 <none> 80:30080/TCP,443:30443/TCP 5h13m
[root@cortx-ova ~]# kubectl describe svc cortx-server-0 -n cortx |grep NodePort:
NodePort: rgw-http 30080/TCP
NodePort: rgw-https 30443/TCP
[root@cortx-ova ~]# endpoint_url="http://""$(kubectl get svc -n cortx | grep cortx-server-0 | awk '{ print $3 }')"":80"
[root@cortx-ova ~]# aws configure set plugins.endpoint awscli_plugin_endpoint
[root@cortx-ova ~]# aws configure set aws_access_key_id sgiamadmin
[root@cortx-ova ~]# aws configure set aws_secret_access_key ldapadmin
[root@cortx-ova ~]# aws configure set s3.endpoint_url $endpoint_url
[root@cortx-ova ~]# aws configure set s3api.endpoint_url $endpoint_url
[root@cortx-ova ~]# echo $endpoint_url
http://10.99.255.213:80

[root@cortx-ova ~]# cat /root/.aws/config
[plugins]
endpoint = awscli_plugin_endpoint
[default]
s3 =
 endpoint_url = http://10.99.255.213:80
s3api =
 endpoint_url = http://10.99.255.213:80

[root@cortx-ova ~]# aws s3 mb s3://mybucket

[root@cortx-ova ~]# dd if=/dev/zero of=file10Mb bs=10MB count=1
1+0 records in
1+0 records out
10000000 bytes (10 MB) copied, 0.00994791 s, 1.0 GB/s

[root@cortx-ova ~]# aws s3 cp file10Mb s3://mybucket/file10Mb
upload: ./file10Mb to s3://mybucket/file10Mb

[root@cortx-ova ~]# aws s3 ls s3://mybucket
2022-09-07 10:55:42 10000000 file10Mb

[root@cortx-ova ~]# aws s3 rm s3://mybucket --recursive
delete: s3://mybucket/file10Mb

[root@cortx-ova ~]# aws s3 rb s3://mybucket
remove_bucket: mybucket

[root@cortx-ova ~]# curl --request POST "https://$IPADDRESS:31169/api/v2/login?debug" --header 'Content-Tycation/json' -d '{"username":"cortxadmin","password":"Cortxadmin@123"}' -k
{"reset_password": false}
```

Test-1 (After reboot):

```
[root@cortx-ova ~]# last reboot |head -2
reboot   system boot  3.10.0-1160.76.1 Thu Sep  8 07:52 - 07:54  (00:01)
reboot   system boot  3.10.0-1160.76.1 Thu Sep  8 07:36 - 07:52  (00:16)

[root@cortx-ova ~]# kubectl get pods -n cortx
The connection to the server 172.21.128.140:6443 was refused - did you specify the right host or port?
WAIT FOR 10 - 15mins AS PER OVA INSTRUCTIONS

[root@cortx-ova ~]# kubectl get pods -n cortx
NAME                             READY   STATUS    RESTARTS      AGE
cortx-consul-client-2hds6        1/1     Running   2 (23m ago)   26h
cortx-consul-server-0            1/1     Running   2 (23m ago)   26h
cortx-control-6bb49c5b68-z88h8   1/1     Running   2 (23m ago)   26h
cortx-data-g0-0                  3/3     Running   6 (23m ago)   26h
cortx-ha-65494bd5c9-fpxgv        3/3     Running   6 (23m ago)   26h
cortx-kafka-0                    1/1     Running   3 (20m ago)   26h
cortx-server-0                   2/2     Running   6 (19m ago)   26h
cortx-zookeeper-0                1/1     Running   2 (23m ago)   26h
[root@cortx-ova ~]#
```

### Changes
 - [ ] Why is this change required? What problem does it solve?
 - [ ] If proposing a new change then please raise an issue first

### How Has This Been Tested? (Optional)
 - [x] Please describe in detail how you tested your changes.
 - [ ] Include details of your testing environment, and the tests you ran to
 - [ ] How your change affects other areas of the code, etc.

### Checklist
 - [x] tested locally
 - [ ] added new dependencies
 - [x] updated the docs
 - [ ] added a test

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -40,3 +40,6 @@
 [submodule "cortx-halo"]
 	path = cortx-halo
 	url = https://github.com/Seagate/cortx-halo.git
+[submodule "cortx-cc"]
```
Comment (shailesh-vaidya) [inline .gitmodules:43]: Please remove this from PR

**Review by shailesh-vaidya — CHANGES_REQUESTED** [review]

File: .gitmodules (hunk 1 above)
Comment (mukul-seagate11) [reply .gitmodules]: removed

**Review by shailesh-vaidya — APPROVED** [review]

**Comment by hessio** [comment]
Can we please also test that the OVA is working from a remote machine also before merging the new release? One of the issues we mentioned for this new OVA was that the ports for the OVA be more inline with industry standard (http: 80 and https: 443)

**Comment by mukul-seagate11** [comment]
@hessio, Its actually not in the requirement as current OVA designed & build for 1N with 1 S3 instance

**Pull request merged**